SearchOC.py

```diff
@@ -0,0 +1,51 @@
+import jax
+import jax.numpy as jnp
+from jax import jit
+
+# start by defining a function to propagate vehicle dynamics (longitudinal)
+
```

Comment:
Include link to paper in comments and refer to equations for relevant lines of code.